backend/src/main/java/co/inventorsoft/scripty/entity/User.java

```diff
@@ -0,0 +1,36 @@
+package co.inventorsoft.scripty.entity;
+import lombok.*;
+import javax.persistence.*;
+import java.util.List;
+
+@Data
```

Comment:
`@Data` annotation generates a lot of stuff. The most dangerous ones are: `toString()`, `equals(Object)` and `hashCode()`.
1) Problems with `toString()`: we don't want password field to be included in generated string. By default `toString()` will include all the fields inside class, including hibernate associations. Therefore when you invoke `toString()` (for example to do logging) it will brake Hibernate's lazy loading mechanism and will fetch more data than you need or expect;
2) Problems with `equals(Object)` and `hashCode()`: in case of entities those methods should be based on primary key(which is id in this case).